gateway/server.go

```diff
@@ -237,6 +239,12 @@ func (gw *Gateway) MarshalJSON() ([]byte, error) {
 	return json.Marshal(struct{}{})
 }
 
+func (gw *Gateway) InitializeRPCCache() {
+	conf := gw.GetConfig()
+	gw.RPCGlobalCache = cache.New(time.Duration(conf.SlaveOptions.RPCGlobalCacheExpiration)*time.Second, 15*time.Second)
```

Comment:
@buger the cleanupInterval i still 15 seconds. I don't think we should make it configurable. However if required just let me know